### DISCUSSION TITLE
LilleSky (UNIT3D)

### DISCUSSION THREAD

**Discussion by bulletproof2k** [autobrr/autobrr#1437] [General]
I am not the owner of this tracker, but I would like it to be in the brr 😊

Tracker Name: [LilleSky.org](http://lillesky.org/)
Genre: General

**Comment by zze0s** [comment]
Hey! We have a indexer request template you can use if you click the New Issue button, or [this link](https://github.com/autobrr/autobrr/issues/new?assignees=&labels=Feature+request%2C+indexers&projects=&template=indexer-request.md&title=%5BINDEXER+REQ%5D).

It has all the required info we need, and if it does not have IRC then we don't add those because you can use them as Generic RSS/Torznab. If they don't have IRC open a thread in their forums and hope they are open to it.

**Comment by bulletproof2k** [comment]
Ok, thanks.
